fenix/app/src/androidTest/java/org/mozilla/fenix/ui/robots/ComposeTabDrawerRobot.kt

```diff
@@ -87,6 +91,30 @@ class ComposeTabDrawerRobot(private val composeTestRule: HomeActivityComposeTest
         composeTestRule.dropdownMenuItemTabSettings().assertExists()
     }
 
+    fun verifyTabClosedSnackbar() {
+        verifySnackBarText("Tab closed")
+    }
+
+    private fun verifySnackBarText(expectedText: String) {
+        TestCase.assertTrue(
+            mDevice.findObject(
+                UiSelector().text(expectedText),
+            ).waitForExists(TestAssetHelper.waitingTime),
+        )
+    }
+
+    fun snackBarUndoButtonClick() {
+        val snackBarButton =
+            mDevice.findObject(
+                UiSelector()
+                    .resourceId("${TestHelper.packageName}:id/snackbar_btn")
+                    .text("UNDO"),
+            )
+
+        snackBarButton.waitForExists(TestAssetHelper.waitingTime)
+        snackBarButton.click()
```

Comment:
You can find both of these methods for the snackbar in TestHelper: https://github.com/mozilla-mobile/firefox-android/blob/main/fenix/app/src/androidTest/java/org/mozilla/fenix/helpers/TestHelper.kt#L131

And you don't need to add the verifyTabClosedSnackbar for this specifically.